src/main/java/cs361/battleships/models/Board.java

```diff
@@ -36,6 +37,131 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
 		} else return false;
 	}
 
+	public void moveShips(String direction)
```

Comment:
All this looks pretty good, it's readable and has logic broken into a different function. Looks pretty good and appears to work correctly.